clippy_lints/src/doc.rs

```diff
@@ -609,3 +694,43 @@ fn check_word(cx: &LateContext<'_>, word: &str, span: Span) {
         );
     }
 }
+
+struct FindPanicUnwrap<'a, 'tcx> {
+    lcx: &'a LateContext<'tcx>,
```

Comment:
NIT: we usually abbreviate both `LateContext` and `EarlyContext` with just `cx` in Clippy.
```suggestion
    cx: &'a LateContext<'tcx>,
```